ui/changes/src/index.js

```diff
@@ -523,6 +553,13 @@ async function buildTable(rerender = true) {
       bugSummary["whiteboard"].includes(whiteBoard)
     );
   }
+  if (releaseVersions) {
+    bugSummaries = bugSummaries.filter((bugSummary) => {
+      return releaseVersions.some(
```

Comment:
Nit: you can avoid the `return` statement if you remove the `{` parenthesis from the arrow function body